examples/exporting_for_inference.ipynb

```diff
@@ -347,6 +400,26 @@
    "source": [
     "`torch.fx` is powerful, but one of the key limitations of this method is that it does not support dynamic control flow (e.g. `if` statements or loop that are data-dependant). Therefore, some algorithms, such as BlurPool, are currently not supported. We have ongoing work to bring `torch.fx` support to all our algorithms."
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9d58df1b",
+   "metadata": {},
+   "source": [
+    "## Algorithm compatibility\n",
+    "Some of our algorithms alter the model architecture in ways that may render them incompatible with some of the export procedures above. For example, BlurPool replaces some instances of `Conv2d` with `BlurConv2d` layers which are not compatible with `torch.fx` as they have data-dependant control flow. \n",
+    "\n",
+    "The following table shows which algorithms are compatible with which export formats for inference.\n",
+    "\n",
+    "|                        | torchscript | torch.fx | ONNX |\n",
+    "|------------------------|-------------|----------|------|\n",
+    "| apply_blurpool         | &check;           |          | &check;    |\n",
```

Comment:
`&check;` is not rendering as a checkbox for me when I use GH's "file view" mode, it shows the text "&check;". Just a suggestion to double check this and fix if needed.

Reply:
It renders well here. https://mosaicml-composer--1360.com.readthedocs.build/en/1360/examples/exporting_for_inference.html

Docs build from this PR.